pyscriptjs/src/main.ts

```diff
@@ -242,7 +254,28 @@ from pyscript import micropip, Element, console, document`);
         for (let i = 0; i < paths.length; i++) {
             logger.info(`  fetching path: ${fetchPaths[i]}`);
             try {
-                await runtime.loadFromFile(paths[i], fetchPaths[i]);
+                const filePromise = fetch(fetchPaths[i]);
+                filePromise.then(async response => {
+                    if(response.ok)
+                    {
+                        const buffer = await response.arrayBuffer();
+                        const data = new Uint8Array(buffer);
+                        if(runtime.isInterpreterReady)
+                        {
+                            runtime.writeToFile(paths[i], data);
+                        }
+                        else
+                        {
+                            this._filesToCopy.push({
+                                path: paths[i],
+                                content: data
+                            })
+                        }
+                    }
+                    else {
+                        throw `💥 Cannot load file from "${fetchPaths[i]}"`;
+                    }
```

Comment:
Since the robustFetch handles any non 200 status code, this logic will always return true because `response.ok` is set on any status 200-299.

As a side note, robustFetch probably shouldn't error on any status > 200 < 300 🤔